resources/lang/de_DE/additional.po

```diff
@@ -27,7 +27,10 @@ msgid "validation.email.email"
 msgstr "Die E-Mail-Adresse ist nicht gültig."
 
 msgid "validation.password.min"
-msgstr "Dein angegebenes Passwort ist zu kurz."
+msgstr "Dein neues Passwort ist zu kurz."
```

Comment:
That also affects the registration pssword but as it should be never used (the form has the minlnength requirement) the wording is not that important here :shrug: